packages/core/src/discovery.js

```diff
@@ -247,6 +279,7 @@ async function* captureSnapshotResources(page, snapshot, options) {
         yield page.evaluate(execute?.beforeResize);
         yield waitForDiscoveryNetworkIdle(page, discovery);
         yield resizePage(width = widths[i + 1]);
+        if (snapshot.multiDOM) { yield page.goto(snapshot.url, { cookies }); }
```

Comment:
This creates a problem when url has a `#id` in it. This is an existing problem with mobile: true as well. Please update both codes to first load `about:blank` and then load the actual url.